frontend/src/metabase/dashboard/actions/data-fetching.js

```diff
@@ -98,9 +98,8 @@ function isNewAdditionalSeriesCard(card, dashcard) {
 
 const updateLoadingTitle = createThunkAction(
   SET_DOCUMENT_TITLE,
-  () => (dispatch, getState) => {
+  totalCards => (_dispatch, getState) => {
     const loadingDashCards = getLoadingDashCards(getState());
-    const totalCards = loadingDashCards.dashcardIds.length;
```

Comment:
The only reason `dashcardIds` exist in our redux store is to be used here. I think removing it would make our dashboard reducer easier to understand.